app/views/unattended/provisioning_templates/snippet/puppet_setup.erb

```diff
@@ -137,15 +119,12 @@ run-puppet-in-installer to true.
 
 Note, that this is an *unsupported mode of operation* and not supported by Foreman at all. You have been warned!
 %>
-<% if host_param_true?('run-puppet-in-installer') -%>
 <% if (os_name == 'Ubuntu' && os_major >= 15) || (os_name == 'Debian' && os_major >= 8) -%>
 # Puppet tries to detect the init system by checking the presence of the directory /run/systemd/system. That detection
-# fails in a chroot environment like the one the installer provides. See Puppet tickets PA-136 and PUP-5577
-#
-# We work around that here until it gets fixed in Puppet (probably never for Puppet 3.x)
+# fails in a chroot environment like the one the installer provides. See Puppet tickets PA-136, PUP-10963 and PUP-5577
+# We work around that here until it gets fixed in Puppet
 mkdir -p /run/systemd/system
```

Comment:
Once `puppet ssl bootstrap` is used, this workaround is only needed for `run-puppet-in-installer`.